neo/Consensus/ConsensusService.cs

```diff
@@ -500,10 +504,10 @@ protected override void OnReceive(object message)
             }
         }
 
-        private void SendChangeViewToRequestRecovery(byte viewNumber)
+        private void RequestRecovery()
         {
```

Comment:
I believe that it would also be nice to have log here, for better identifying who is asking for recovery in the historical logs.